src/NServiceBus.Testing/Saga.cs

```diff
@@ -6,6 +6,11 @@
     using System.Threading.Tasks;
     using MessageInterfaces.MessageMapper.Reflection;
 
+    internal static class SagaConsts
```

Comment:
because the saga is a generic type and you would have to specify a generic param to access the constant